### ISSUE TITLE
Make Lantern Error on macOS Sierra. Log Here

### ISSUE THREAD

**Issue by zhangtianyu2012** [getlantern/lantern#5272]
```
Building development lantern
** VERSION was not set, using default version. This is OK while in development.
Build tags:
Extra ldflags:
# github.com/getlantern/flashlight/proxied
src/github.com/getlantern/flashlight/proxied/proxied.go:391: tr.MaxIdleTime undefined (type *http.Transport has no field or method MaxIdleTime)
src/github.com/getlantern/flashlight/proxied/proxied.go:392: tr.EnforceMaxIdleTime undefined (type *http.Transport has no field or method EnforceMaxIdleTime)
# github.com/getlantern/flashlight/client
src/github.com/getlantern/flashlight/client/reverseproxy.go:20: unknown http.Transport field 'MaxIdleTime' in struct literal
make: *** [lantern] Error 2
```

[log.txt](https://github.com/getlantern/lantern/files/466463/log.txt)

**Comment by ACookieBear** [comment]
I meet the same problem and resolved just now
edit the source code `lantern/src/github.com/getlantern/flashlight/proxied/proxied.go` and `lantern/src/github.com/getlantern/flashlight/client/reverseproxy.go`, change the "MaxIdleTime" to "IdleConnTimeout"and delete "transport.EnforceMaxIdleTime()"

**Comment by zhangtianyu2012** [comment]
@ACookieBear Thanks, I build seccess. But I can't open the website like [Google](http://www.google.com) or [Facebook](http://www.facebook.com) after I execute `./lantern`. And the form of the lantern homepage looks like the version of 2.x. It doesn't have a login entrance.

**Comment by ACookieBear** [comment]
对啊是老版的，我也挺郁闷的，我用./lantern能打开google，但是不太稳定，不知道哪个分支是专业版的源码，如果没有放出来只能用wine跑windows版的了，不知道为啥不发行个rpm版

**Comment by Artoria2e5** [comment]
> 不知道为啥不发行个rpm版

够浪的二进制经常可以到处拿着跑（不静态都经常可以），考虑解包一下？

**Comment by p3rcya** [comment]
新版蓝灯3.2已经支持邮箱登陆和MacOS Sierra，请大家去论坛主页 https://github.com/getlantern/forum 下载。感谢各位的耐心等待！

**Comment by fanyer** [comment]
same here and solved MacOS  10.11.6 @ACookieBear  THX
